phockup.py

```diff
@@ -149,50 +173,105 @@ def main():
 
 To get all date fields available for a file, do:
     exiftool -time:all -mimetype -j <file>
-        """,
+""",
     )
 
+    exclusive_group_debug_silent = parser.add_mutually_exclusive_group()
 
-    parser.add_argument(
-        "-q",
+    exclusive_group_debug_silent.add_argument(
+        "--debug",
+        action="store_true",
+        default=False,
+        help="""\
+Enable debugging.
```

Comment:
Same here. I don't like this taking 3 lines instead of 1. Please let me know if this is PEP8 requirement.

Reply:
It's not a PEP8 requirement for PEP8 to do it like this as long it's within 79 characters per line. I thought everywhere in the code to keep the same formatting.